.github/workflows/promptflow-sdk-cli-test.yml

```diff
@@ -14,30 +15,51 @@ env:
   PYTHONPATH: ${{ github.workspace }}/src/promptflow
   IS_IN_CI_PIPELINE: "true"
 jobs:
-  authorize:
-    environment:
-      # forked prs from pull_request_target will be run in external environment, domain prs will be run in internal environment
-      ${{ github.event_name == 'pull_request_target' &&
-      github.event.pull_request.head.repo.full_name != github.repository &&
-      'external' || 'internal' }}
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        pythonVersion: ['3.8', '3.9']
```

Comment:
why we need two python version for build stage?

Reply:
your code may have build issues across python versions.
For example

Task[Any] is not support in python 3.8, but is working in python 3.9.

3.11 is not stable, I'm working on 3.11 on another PR.

Please use a version compatible way in coding.

Reply:
if so, I think we should open bug for the code author of `Task[Any]`?

Reply:
and if we have two version here, what's behavior when consume? the later stage will download/consume which wheel?

Reply:
any wheel is OK, wheel itself is cross platform and cross version.

I use 3.9, you can see it in below